docs/types.rst

```diff
@@ -290,12 +293,15 @@ including public state variables.
 Fixed-size byte arrays
 ----------------------
 
-``bytes1``, ``bytes2``, ``bytes3``, ..., ``bytes32``. ``byte`` is an alias for ``bytes1``.
+The value types ``bytes1``, ``bytes2``, ``bytes3``, ..., ``bytes32``
+can be used to hold a sequence of bytes from one to up to 32.
```

Comment:
@chriseth You could switch out the passive voice here if you wanted to 😀

Reply:
Can you give an example?

Reply:
Ah, I just remove the "can be used to"